ext/node/polyfills/internal_binding/_utils.ts

```diff
@@ -88,6 +88,102 @@ export function utf16leToBytes(str: string, units?: number) {
   return i * 2 === length ? byteArray : byteArray.subarray(0, i * 2);
 }
 
+export function utf8ToBytes(str: string, units?: number) {
```

Comment:
Can we use the text encoding APIs for this?

Reply:
I was thinking about that as well. I'm not quite sure is the answer :)

Reply:
It was indeed possible.